var/spack/repos/builtin/packages/atlas/package.py

```diff
@@ -51,6 +51,7 @@ class Atlas(Package):
             url='http://sourceforge.net/projects/math-atlas/files/Developer%20%28unstable%29/3.11.34/atlas3.11.34.tar.bz2')
 
     variant('shared', default=True, description='Builds shared library')
+    variant('openmp', default=False, description='Use multithreaded libraries')
```

Comment:
for convenience, copy-paste from the comment which is hidden now:

the threaded variant should NOT be called openmp, atlas by default uses `pthreads`, but can be forced to use `openmp` http://math-atlas.sourceforge.net/errata.html:

> Building using OpenMP: In general, this is a bad idea, since OpenMP tends to much slower than pthreads for ATLAS use. However, if your own application uses OpenMP, sometimes pthread usage can slow down your own threads, making it worthwhile to damage ATLAS performance in order to improve your OpenMP performance. In ATLAS, thread affinity is the main reason pthreads wins against OpenMP, and so OS~X (and probably FreeBSD), which don't support real affinity, are the platforms where it makes sense to use OpenMP regardless. To force ATLAS to use OpenMP rather than pthreads, you must add the following flags to your configure line:
-Si omp 1 -F alg -fopenmp

Maybe call it `multithreaded` ?